site/docs/components/card/examples.mdx

```diff
@@ -22,7 +22,7 @@ By default, the card component does not have any interactive or focusable elemen
 
 ## Variants
 
-All card components provide primary, secondary and tertiary variants, offering flexibility and enabling differentiation based on visual hierarchy within your UI.
+All card components provide primary, secondary, tertiary and alpha variants, offering flexibility and enabling differentiation based on visual hierarchy within your UI.
```

Comment:
Does alpha deserve it's own explanation?
Something like, if you use "alpha" the visual hierarchy is then defined by it's parent ?